<ui version="4.0" >
 <class>FindAnnotationCollocationsDialog</class>
 <widget class="QDialog" name="FindAnnotationCollocationsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>dialog_tittle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QTreeWidget" name="annotationsTree" >
           <property name="minimumSize" >
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="itemsExpandable" >
            <bool>false</bool>
           </property>
           <property name="headerHidden" >
            <bool>true</bool>
           </property>
           <property name="columnCount" >
            <number>2</number>
           </property>
           <column>
            <property name="text" >
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>2</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="1" >
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" >
          <widget class="QListWidget" name="resultsList" >
           <property name="minimumSize" >
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <layout class="QHBoxLayout" name="horizontalLayout" >
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>region_size_label</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="regionSpin" >
             <property name="minimumSize" >
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum" >
              <number>10</number>
             </property>
             <property name="maximum" >
              <number>100000000</number>
             </property>
             <property name="value" >
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="searchButton" >
             <property name="text" >
              <string>search_button</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2" >
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="saveResultsButton" >
             <property name="text" >
              <string>save_annotations_button</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearResultsButton" >
             <property name="text" >
              <string>clear_results_button</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QCheckBox" name="wholeAnnotationsBox" >
           <property name="text" >
            <string>whole annotations</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>228</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton" >
           <property name="text" >
            <string>cancel_button</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="statusBar" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
